### PR TITLE
Add a github pull request template

### PR DESCRIPTION
**Pull request by kroeckx**

### CODE REVIEW

**Comment by kroeckx** [comment]
I'm wondering if this actually works, I didn't see the template myself.

**Comment by levitte** [comment]
Well, it wasn't in the repo when you did the pull request, so that doesn't surprise me...

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!--
+Thank you for your pull request. Please review below requirements.
+
+Contributors guide: https://github.com/openssl/openssl/blob/master/CONTRIBUTING
+-->
+
+##### Checklist
+<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->
+- [ ] documentation is changed or added
+- [ ] tests are updated
```
Comment (levitte) [inline .github/PULL_REQUEST_TEMPLATE.md]: Should there be something about a CLA here as well?

Comment (kroeckx) [inline .github/PULL_REQUEST_TEMPLATE.md]: I added it at some point, but I removed it again.

Comment (kroeckx) [inline .github/PULL_REQUEST_TEMPLATE.md]: So I added it.

**Comment by levitte** [comment]
Ok, go

**Comment by kroeckx** [comment]
It seems to show up now.

**Pull request closed without merging**